sklearn/impute.py

```diff
@@ -965,3 +978,225 @@ def fit(self, X, y=None):
         """
         self.fit_transform(X)
         return self
+
+
+class MissingIndicator(BaseEstimator, TransformerMixin):
+    """Binary indicators for missing values.
+
+    Parameters
+    ----------
+    missing_values : number, string, np.nan (default) or None
```

Comment:
what does number mean and why is np.nan not a number? Maybe just move the np.nan to the end?

Reply:
number means real number. It's just to fit this in one line. 
I think by definition nan is not a number :)

Reply:
but the dtype is also important, isn't it? I find "float or int" more natural than "number or np.nan" but I don't have a strong opinion.

Reply:
I agree that "float or int" is better than number, but I think it's important to keep np.nan visible since it should be a common value for `missing_values`. Maybe something like
int, float, string or None (default=np.nan) ?

Reply:
right now this is consistent with SimpleImputer and ChainedImputer in fact.